spring-data-rest/src/test/java/org/baeldung/relationships/SpringDataRelationshipsTest.java

```diff
@@ -0,0 +1,103 @@
+package com.baeldung.relationships;
```

Comment:
based on the package name, the file should be in **com**/baeldung/relationships instead of **org**/baeldung/relationships